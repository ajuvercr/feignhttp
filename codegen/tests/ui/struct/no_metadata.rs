use feignhttp_codegen::feign;

// error: no metadata assign
//    |  #[feign]
//    |  ^^^^^^^^

struct Http;

#[feign]
impl Http {}

fn main() {}
