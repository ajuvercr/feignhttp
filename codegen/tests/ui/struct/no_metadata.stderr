error: no metadata assign
 --> tests/ui/struct/no_metadata.rs:9:1
  |
9 | #[feign]
  | ^^^^^^^^
  |
  = note: this error originates in the attribute macro `feign` (in Nightly builds, run with -Z macro-backtrace for more info)
